CHANGELOG.md

```diff
@@ -193,7 +194,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## Version 1.0.0 - 2014-08-13
 Initial release.
 
-[unreleased]: https://github.com/contentful/contentful.java/compare/java-sdk-8.0.0...HEAD
+[unreleased]: https://github.com/contentful/contentful.java/compare/java-sdk-8.1.0...HEAD
+[8.1.0]: https://github.com/contentful/contentful.java/compare/java-sdk-8.0.0...java-sdk-8.1.0
```

Comment:
I'm a bit confused, why is 8.1.0 marked as unreleased? isn't the plan to release it on merge of this PR?

Reply:
The 8.1.0 is not yet released. Before I do the releasing I'll test it manually and add a date and remove the flag.